addons/survey/views/survey_question_views.xml

```diff
@@ -170,6 +170,11 @@
                                     <!-- Char -->
                                     <field name="validation_length_min" attrs="{'invisible': ['|', ('question_type', '!=', 'char_box'), ('validation_required', '=', False)]}"/>
                                     <field name="validation_length_max" attrs="{'invisible': ['|', ('question_type', '!=', 'char_box'), ('validation_required', '=', False)]}"/>
+
+                                    <field name='comments_allowed' attrs="{'invisible':[('question_type','not in',['simple_choice','multiple_choice', 'matrix'])]}"/>
+                                    <field name='comments_message'
+                                        attrs="{'invisible': ['|', ('question_type', 'not in', ['simple_choice','multiple_choice', 'matrix']), ('comments_allowed', '=', False)]}"/>
```

Comment:
Isn't it a bit strange that these fields are lost between "Char" and "Numerical" validation sections?

Could be put either at the top or at the bottom of this `<group>` maybe?
I would vote for just above "question_placeholder" or above "matrix_subtype", depending on where we want to see those appear (maybe double-check with LNA?).

Reply:
Yes you're right. The only thing important is that it belongs to the group answer. I have put it at the end of that group but wil check with Luc for the order (in one case this field is visible with an other).